### DISCUSSION TITLE
Why did you write the Content Fake build using only createProcess?

### DISCUSSION THREAD

**Discussion by BennieCopeland** [SAFE-Stack/SAFE-template#597] [Q&A] [answered]
I'm looking at this project for inspiration on how to use Fake, especially for running backend and frontend tests in parallel. I notice that the script isn't using any of the helpers like Fake.DotNet or Fake.JavaScript but instead have a helper function to create a process. I'm curious for the reason for this. Personal preference, technical reasons, something else?

**Comment by theprash** [comment, marked as answer]
Hi Bennie. I didn't make the decision to use this style so I can't speak for whoever did. But it seems to me that there are some benefits of the console command style:

* It's simpler
* It aligns with what you would type to run the commands, so you can easily see how to run them manually
* It's guaranteed to give the same result as typing the command, making it more consistent and predictable. Sometimes the FAKE helpers try to do more helpful things.
